library/config.py

```diff
@@ -144,6 +144,12 @@ def compute(self) -> dict:
                 url = f"https://data.cityofnewyork.us/api/views/{_uid}/rows.csv"
             if _format == "geojson":
                 url = f"https://nycopendata.socrata.com/api/geospatial/{_uid}?method=export&format=GeoJSON"
+            if _format == "shapefile":
+                local_path = f"library/tmp/{config['dataset']['name']}.zip"
+                url = f"https://data.cityofnewyork.us/api/geospatial/{_uid}?method=export&format=Shapefile"
```

Comment:
just out of curiosity, is it significant that the first parts of these URLs are different?

Reply:
That's a good point - I suspect the `data.cityofnewyork.us` sits on top of socrata? If you look at the "csv" option above, it also uses `data.cityofnewyork.us`